apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dice in a cup
+  """
+
+  @spec new(integer) :: [integer]
+  def new(n, acc \\ [])
+  def new(0, acc), do: acc
+  def new(n, acc) do
+    new(n-1, [:rand.uniform(6) | acc])
+  end
```

Comment:
Most of these types of things are in `Stream` because they only work lazily (it's an infinite list of function calls otherwise, which would never end).